scripts/EN/Harry

```diff
@@ -8364,7 +8364,7 @@ Area(0x130); // 0xa9fe: 00001026
 CleanScreen(); // 0xaa04: 00000064
 LoadBackground(0x0); // 0xaa08: 00001069
 DrawBackground(); // 0xaa0e: 00000065
-ShowText((4510)"The nightmare still continues..."); // 0xaa12: 0000100c
+ShowText((4510)"The nightmare continues on..."); // 0xaa12: 0000100c
```

Comment:
Never liked the flow of "still continues" in English so I removed still and added on.